Add recommendation to put hyphen after OCID prefix (#1113)
open-contracting/standard: docs/schema/identifiers.md

The ocid is case-sensitive; in other words, the letter case of an ocid must be consistent.

> It is good practice to separate the ocds prefix and the internal identifier with an hyphen (`-`).

Reviewer: ```suggestion
It is encouraged to separate the ocds prefix and the internal identifier with a hyphen (`-`).
```

Author: Sorry, I had forgotten the documentation.
